content/en/guides/get-started/installation.md

```diff
@@ -12,7 +12,7 @@ ManualInstallVideoTitle: Nuxt Manual Installation
 
 ## Prerequisites
 
-- [node](https://nodejs.org) - _We recommend you have the latest LTS version installed_ or at least v10.13 and above (excluding v11).
+- [node](https://nodejs.org) - _We recommend you have the latest LTS version installed_ or at least v12 and above.
```

Comment:
```suggestion
- [node](https://nodejs.org) - _We recommend you have the latest LTS version installed_.
```

Reply:
OK. That will remove the info about minimum supported version (even if it's not exactly phrased that way). I guess that's better from maintenance point of view.